cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -142,6 +143,16 @@ type NodeConfiguration struct {
 	// without CA verification via DiscoveryTokenCACertHashes. This can weaken
 	// the security of kubeadm since other nodes can impersonate the master.
 	DiscoveryTokenUnsafeSkipCAVerification bool
+
+	// FeatureGates enabled by the user
+	FeatureGates map[string]bool
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
why add this?

Reply:
I was thinking of adding this to make `SetDefaults_KubeletConfiguration` registered.